Pack-Smart/pack-smart-fe
# Feature/update styles

## What's this PR do?
- Creates `HowItWorks` component with `Step` and `Review` helper components
- Updates `Quiz` styles
- Updates `SavedView` styles
- Updates `PackingList` styles
- Updates and fixes error component in `Quiz`

## Where should the reviewer start?
- `HowItWorks.js`, `Step.js`, `Review.js`, `SavedView.js` and `PackingList.js`

## How should this be manually tested?
- Fill out the quiz without completing all required fields to see error component
- Fill out the quiz correctly and see the packing list
- Click on `How It Works`
- Click on `Saved Lists`

## What are the relevant tickets?
- Closes #28 
- Closes #40 
- Closes #42 
- Closes #38 

## Screenshots/GIFs
- <img width="1008" alt="Screen Shot 2021-02-25 at 3 08 37 PM" src="https://user-images.githubusercontent.com/58453566/109226702-6d664100-777c-11eb-8b7e-59f9dbcd19dd.png">
- <img width="1007" alt="Screen Shot 2021-02-25 at 3 08 49 PM" src="https://user-images.githubusercontent.com/58453566/109226731-77883f80-777c-11eb-9c56-61298c453f52.png">
- <img width="848" alt="Screen Shot 2021-02-25 at 3 09 13 PM" src="https://user-images.githubusercontent.com/58453566/109226743-7bb45d00-777c-11eb-9ac6-b4e555ef386b.png">

## Questions:
- Any input on home page styles? The rest of the site is feeling very cohesive to me!

## Did you update the README?
- Not yet!
